packages/css/src/shorthand-parser/index.ts

```diff
@@ -4,6 +4,10 @@ import { tokenizeValue } from "./value-tokenizer";
 const unitMatch = /^[0-9.]+[a-z|%]/;
 const easingMatch = /\(.*\)|ease|ease-in|ease-out|ease-in-out|linear|step-start|step-end/;
 
+const fontSizeMatch = /^([+-]?[\d.]+([a-z]+|%)?|large(r)?|medium|small(er)?|x{1,3}-large|x{1,2}-small)(\/[+-]?[\d.]+([a-z]+|%)?)?$/;
+const fontStyleMatch = /^[+-]?[\d.]+deg$/;
+const fontWeightMatch = /^(0*[1-9]\d{0,2}|1000|bold(er)?|lighter)$/;
```

Comment:
I only now notice that this project is using `[0-9]`, while I am using `\d`. Let me know if I should adjust these. These regular expressions are otherwise intended to follow the patterns seen in `unitMatch` and `easingMatch`.

Note: A more accurate regular expression for CSS numbers may be `/([+-]?(?:\d+(?:\.\d*)?|\.\d+)(?:[Ee][+-]?\d+)?)/`. However, if parsing performance is of importance, it may be better to identify numbers at the tokenization stage and leave out regular expressions altogether.